incubator/redis-auto-cluster/templates/config.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-config
```

Comment:
Resource name should be based on the fullname partial. Apply throughout the chart.